integrations/standalone/tests/pageobjects/FieldMessage.ts

```diff
@@ -0,0 +1,18 @@
+import { expect, type Locator } from '@playwright/test';
+
+export class FieldMessage {
+  readonly locator: Locator;
+
+  constructor(parentLocator: Locator, options: { label: string }) {
+    this.locator = parentLocator.getByLabel(options.label, { exact: true }).locator('.ui-message');
+  }
+
+  async expectToBeHidden() {
+    await expect(this.locator).toBeHidden();
+  }
+
+  async expectToHaveErrorMessage(message: string) {
+    await expect(this.locator).toHaveText(message);
+    await expect(this.locator).toHaveAttribute('data-state', 'error');
+  }
```

Comment:
e.g. this makes sense to me.
But the `expectToBeHidden` method is pretty useless, the test code can also write `await expect(locator).toBeHidden()`